### PR TITLE
Add disease field to event creation modal

### PR DESCRIPTION
**Pull request by nathanathan**

### CODE REVIEW

File: client/controllers/events/createEventModal.coffee [hunk 1]
```diff
@@ -3,17 +3,20 @@
...
     unless valid
       toastr.error('Please specify a valid name')
       eventName.focus()
       return
-    newEvent = eventName.value
-    summary = target.eventSummary.value
-
-    Meteor.call 'editUserEvent', null, newEvent, summary, (error, result) ->
+    Meteor.call 'upsertUserEvent', {
+      eventName: eventName.value.trim()
```
Comment (Broham) [inline client/controllers/events/createEventModal.coffee]: Do we want to move this `trim` to line 8 for consistency sake?

File: imports/schemas/userEvent.coffee [hunk 2]
```diff
@@ -4,22 +4,32 @@ userEventSchema = new SimpleSchema(
     optional: true
   articleCount:
     type: Number
+    optional: true
```
Comment (Broham) [inline imports/schemas/userEvent.coffee]: [I believe that](https://github.com/aldeed/meteor-simple-schema#optional) `optional: true` is the default in simple schema, are all of these required?

File: methods/userEvents.coffee [hunk 3]
```diff
@@ -1,27 +1,26 @@
...
-            disease: disease
-            lastModifiedDate: now
-            lastModifiedByUserId: user._id
-            lastModifiedByUserName: user.profile.name
-          $setOnInsert:
-            creationDate: now
-            createdByUserId: user._id
-            createdByUserName: user.profile.name
-            articleCount: 0
+  upsertUserEvent: (userEvent) ->
```
Comment (Broham) [inline methods/userEvents.coffee]: I noticed that before we were trimming name, before we upserted.  Just wanted to confirm that we no longer need to do this.

File: imports/schemas/userEvent.coffee (hunk 2 above)
Comment (nathanathan) [inline imports/schemas/userEvent.coffee]: The default is for them to be required. The keys are not required by the meteor method using this schema for validation.

File: methods/userEvents.coffee (hunk 3 above)
Comment (nathanathan) [inline methods/userEvents.coffee]: I moved the trimming to the call side.

File: imports/schemas/userEvent.coffee (hunk 2 above)
Comment (Broham) [inline imports/schemas/userEvent.coffee]: Oh, you are right, I totally misread that, sorry

File: methods/userEvents.coffee (hunk 3 above)
Comment (Broham) [inline methods/userEvents.coffee]: ok sounds good

**Comment by Broham** [comment]
Looks good :+1:

**Pull request merged**